README.md

```diff
@@ -256,18 +256,18 @@ await mcpManagerDebug.initialize();
 // MCP_DEBUG=1 codex
 ```
 
-#### Using the HTTP Server API
+#### Using the HTTPmcpServer API
 
 You can expose MCP functionality over HTTP:
 
 ```typescript
-// Start an HTTP server on port 3000
+// Start an HTTPmcpServer on port 3003
```

Comment:
Changes like this make it seem you may have just done a find and replace for the word " server" and replace with "mcpServer".  This change looks unintentional.